### DISCUSSION TITLE
TypeError: Required parameter edge_index is empty.

### DISCUSSION THREAD

**Discussion by LLLLucensus** [pyg-team/pytorch_geometric#8839] [Q&A]
It seems to be the internal error torch_geometric. Do anybody know the solution or tricks to walk around it?
>     m1=self.propagate(edge_index=edge_index1,size=(x1.size(0), x1.size(0)), x=x1,edge_weight=edge_weight1)
  File "/opt/conda/lib/python3.7/site-packages/torch_geometric/nn/conv/message_passing.py", line 257, in propagate
    msg_kwargs = self.__distribute__(self.__msg_params__, kwargs)
  File "/opt/conda/lib/python3.7/site-packages/torch_geometric/nn/conv/message_passing.py", line 180, in __distribute__
    raise TypeError(f'Required parameter {key} is empty.')
TypeError: Required parameter edge_index is empty.

**Comment by rusty1s** [comment]
I cannot reproduce this. Do you mind to share a reproducible example? Can you also try to call via `self.propagate(edge_index, size=...)` (not using `edge_index` as a keyword argument)?
